_docs/quickstarts/scanner/descriptions.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Scanner Specifications
+order: 102
+---
+
+# {{ page.title }}
+
+This page describes the Forseti scanners that are available, how they work, and
+why they're important. You can configure Scanner to execute multiple scanners in
+the same run. Learn about [configuring
+Scanner]({% link _docs/configure/scanner/index.md %}).
+
+## BigQuery dataset ACL scanner
+
+BigQuery datasets have access properties that can publicly expose your datasets.
+The BigQuery scanner supports a blacklist mode, to ensure unauthorized users
+don't gain access to your datasets.
+
+For examples of how to define scanner rules for your BigQuery datasets, see the
+[`bigquery_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/rules/bigquery_rules.yaml)
+rule file.
+
+## Blacklist scanner
+
+VM instances with external IP addresses communicate with the outside world.
+If they are compromised, they could appear in various 
+blacklists and could be known as malicious, such as for sending spam, 
+hosting Command & Control servers, and so on. The blacklist scanner audits
+all of the VM instances in your environment and determines if any VMs
+with external IP addresses are on a specific blacklist you've configured.
+
+For examples of how to define scanner rules, see the
+[`blacklist_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/rules/blacklist_rules.yaml)
```

Comment:
feel free to let URL lines extend pass 80

Reply:
Updated.